Mathlib/AlgebraicTopology/TopologicalSimplex.lean

```diff
@@ -73,15 +73,14 @@ def toTop : SimplexCategory ⥤ TopCat where
     ext f : 1
     apply toTopObj.ext
     funext i
-    change (Finset.univ.filter (· = i)).sum _ = _
+    dsimp
     simp [Finset.sum_filter, CategoryTheory.id_apply]
```

Comment:
Does this work?
```suggestion
    simp [Finset.sum_filter, CategoryTheory.id_apply]
```

Reply:
In fact, the `ext` above can also be cleaned up, thanks for making me notice :)